ARCHITECTURE.md

```diff
@@ -40,12 +40,10 @@ Actions (e.g., the parsing of CSV files, conversion of media files, or storage o
 
 ## Sequence Diagram
 
-The sequence diagram below illustrates the order of events that happen as the av-pairtree program processes an A/V file and puts it into a [Pairtree](https://tools.ietf.org/html/draft-kunze-pairtree-01) directory structure.
+The sequence diagram below illustrates the order of events that happen as the av-pairtree program processes a CSV:
```

Comment:
You don't think it's useful to link to the Pairtree spec?

Reply:
Oh, for some reason I didn't realize I'd removed the link. I do think it's useful but maybe earlier in the README?

Reply:
er, [it's already linked in the "Bird's Eye View" section](https://github.com/UCLALibrary/av-pairtree/pull/10/files#diff-8f6366fd8e7a5fa30f7f05879999195b7cf5fa1e3d157822eb37f0e91d226cfdR9) -- that's enough IMO

Reply:
Okay 👍🏼